packages/storefront-lwc/src/modules/commerce/productSearchResults/productSearchResults.js

```diff
@@ -218,4 +224,20 @@ export default class ProductSearchResults extends LightningElement {
 
         return filtersArray;
     };
+
+    /**
+     * Set the apropriated text in the heading.
+     * @param showHeader A flag indicating whether or not to show the heading
+     */
+    createHeader(showHeader) {
+        if (showHeader && this.hasProducts()) {
+            this.searchtext = 'Search result for';
+            this.headertext = this.query;
```

Comment:
Maybe nitpicky, but what if someone used this component for something that wasn't for search, and still wanted to use `searchtext`? Is there a better word for the property? `headertext` and `secondarytext`? I don't know. The `headertext` is what is on mobile and desktop, whereas the other is what disappears in mobile.

Reply:
renamed to property to headertextprefix.